specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -1841,12 +1850,19 @@ runtimeclass PackageDependency
     /// Calls to Add() can be balanced by a PackageDependencyContext.Remove() (or object destruction)
     /// to remove the entry from the package graph.
     ///
-    /// Successful calls change the package graph's current generation id.
+    /// Successful calls change the package graph's current revision id.
     ///
-    /// @see GenerationId
+    /// @see PackageGraphRevisionId
     PackageDependencyContext Add(AddPackageDependencyOptions options);
 
-    /// Return the package graph's current generation id.
+    /// Return the package graph's current revision id.
+    [contract(DynamicDependencyContract, 2)]
+    static UInt32 PackageGraphRevisionId{ get; };
```

Comment:
Standard guideline is to use Int32, even if signed is invalid, unless the API really needs more than 2B values

Reply:
That's news. This is a wrapper over the Win32 API which returns `uint32`. Still worth going `int32`?

Reply:
Existing functions already return U32, so probably better to just continue with that.